src/main/java/bio/terra/service/rawls/RawlsClient.java

```diff
@@ -58,4 +60,31 @@ public WorkspaceResponse getWorkspace(UUID workspaceId, AuthenticatedUserRequest
   String getWorkspaceEndpoint(UUID workspaceId) {
     return String.format("%s/api/workspaces/id/%s", rawlsConfiguration.basePath(), workspaceId);
   }
+
+  public void verifyBillingProjectAction(
+      UUID billingProjectId, String action, AuthenticatedUserRequest userRequest) {
```

Comment:
Since this is our API you could make it accept `IamAction` 
```suggestion
      UUID billingProjectId, IamAction action, AuthenticatedUserRequest userRequest) {
```

And could also change `verifyBillingProjectActionEndpoint()` below; the action's `toString()` will be called when `String.format()` is called.